app/routes/events.tsx

```diff
@@ -20,18 +23,26 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
 
 export default function EventsRoute() {
 	const { events } = useLoaderData<typeof loader>()
+	const [searchValue, setSearchValue] = useState("")
 
 	return (
-		<div>
-			<section className="section-auto">
-				<header className="space-y-4">
-					<h1>Events</h1>
-					<p>More info coming soon</p>
+		<div className="relative">
+			{/* <div className="spectrum-background" /> */}
+			<GradientBackground />
+
+			<section className="section-auto relative">
+				<header className="space-y-4 pt-16">
+					<h1 className="mb-10 text-center">Event</h1>
+					<SearchInput
+						className="mx-auto block max-w-md"
+						placeholder="Cari event, contoh: Javascript"
+						value={searchValue}
+						setValue={evt => setSearchValue(evt.target.value)}
+					/>
 				</header>
 			</section>
 
-			<section className="section-auto space-y-4">
-				<h2>Public Events</h2>
+			<section className="section-auto relative min-w-full space-y-4">
```

Comment:
I think you can limit the maximum width of the section to overcome if the screen is very wide, such as in the section-auto class. 

```css
.section-auto {
	@apply mx-auto max-w-3xl px-4 py-10;
}
```

You can create a new class like section-auto class and then you change 'max-w-3xl' to bigger than 3xl.

Reply:
I've create a new class for it and I set max width to 6xl

Reply:
Cool